content/daytrip/eu/gb/the-birmingham-man.md

```diff
@@ -3,7 +3,7 @@ slug: daytrip/eu/gb/the-birmingham-man
 date: '2025-05-25T10:15:15'
 lat: '52.4798132'
 lng: '-1.9042805'
-location: 'The Birmingham Man: Thomas Attwood (1783-1856), Chamberlain Square, Chinese Quarter, Ladywood, Park Central, Birmingham, West Midlands, England, B3 3DQ, United Kingdom'
+location: 'The Birmingham Man, Chamberlain Square, Ladywood, Birmingham, West Midlands, B3 3DQ, United Kingdom'
```

Comment:
This is more the kind of edit I'd prefer to see, rather than removing the country names from within the UK.